### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by ooreally**
feat: added wishlist page to Ecom
Here is ss of the page
image

### CODE REVIEW

File: pages/wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+@import url("../../css/index.css"); 
+
+.wishlist-contents{
```
Comment (ckahub) [inline pages/wishlist/wishlist.css:3]: use rem everywhere

**Review by ckahub — COMMENTED** [review]
looks good to merge

File: index.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
                 <div class="hamburger"> <i class="fa-solid fa-bars med-ham "></i></div>
                 <div class="hamburger-dropdown-content">
```
Comment (Asuraking1n) [inline index.html:21]: Give drop down z index more then content. Drop is behind the text

**Review by Asuraking1n — COMMENTED** [review]
Everything else looks cool

**Comment by ooreally** [comment]
Merged pr#4 to add wishlist page

**Pull request merged**